data.sql

```diff
@@ -1,8 +1,40 @@
-INSERT INTO animals (name, date_of_birth, escape_attempts,
-neutered, weight_kg) 
+INSERT INTO vets(name, age, date_of_graduation)
 VALUES 
-('Agumon', '2020-02-03', 0, FALSE,10.23),
-('Gabumon', '2018-11-15', 2, TRUE,8.0),
-('Pikachu', '2021-01-07', 1, FALSE,15.04),
-('Devimon', '2017-05-12', 5, TRUE,11.0)
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '2000-04-23'),
+('Jack Harkness', 38, '2008-06-08')
 RETURNING *;
```

Comment:
- [ ] Kindly make sure the data you insert here is correct, the `date_of_graduation` of the third vets is '1981-05-04' not '2000-04-23'. Kindly correct this. 👍🏻

Reply:
Thank you @BenMukebo; I have rectified the data.